pkg/sql/colexec/agg/bit_and.go

```diff
@@ -59,6 +62,9 @@ func (ba *BitAnd[T1]) Fill(groupIndex int64, v1 T1, v2 uint64, z int64, isEmpty
 	if isEmpty {
 		v2 = ^uint64(0)
 	}
+	if float64(v1) > math.MaxUint64 {
+		return math.MaxInt64 & v2, false
+	}
 	return uint64(v1) & v2, false
```

Comment:
Is this the semantics of mysql. What is the meaning of this a..

Reply:
When x86 and M1 overflow the result of uint64 type conversion, the processing methods are different, which leads to the generation of bugs. Therefore, it is used to directly determine whether the uint64 range is exceeded. If the uint64 range is exceeded, it is calculated according to the maximum value of int64, which is consistent with MySQL